### PR TITLE
Update manifest-base.json

### PR DESCRIPTION
**Pull request by guifarias31**
## Summary of Changes
Updated dependencies version and mininum game version.

## Additional context

Discord username (if different from GitHub): [Guilherme Farias#9971](https://discordapp.com/channels/@me/207215431654572032)

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build.py will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, slick on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

File: manifest-base.json [hunk 1]
```diff
@@ -10,24 +10,24 @@
...
       "name": "asobo-vcockpits-instruments-airliners"
     },
     {
-      "package_version": "0.1.66",
+      "package_version": "0.1.69",
       "name": "fs-base-aircraft-common"
     }
   ],
   "content_type": "AIRCRAFT",
-  "total_package_size": "00000000000221782906",
```
Comment (devsnek) [inline manifest-base.json:30]: you could remove this line entirely, build.py overrides it anyway.

Comment (guifarias31) [reply manifest-base.json:30]: Roger that.

File: manifest-base.json [hunk 2]
```diff
@@ -10,24 +10,24 @@
...
-  "minimum_game_version": "1.9.5",
+  "total_package_size": "00000000000325682532",
+  "minimum_game_version": "1.10.5",
```
Comment (lukecologne) [inline manifest-base.json:31]:
```suggestion
  "minimum_game_version": "1.10.8",
```
Current game version is 1.10.8

**Comment by Benjozork** [comment]
No response.

**Pull request closed without merging**